src/accounts/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from "react";
+
+import Wallet from "./wallet";
+import axios from 'axios';
+import AddWalletModal from "../modals/addWallet";
+import Loader from "../components/shared/Loader";
+import Swal from "sweetalert2";
+
+import "./style.scss"
+
+export default function Accounts() {
+
+    const [accountDB, setAccountDB] = useState([]);
+    const [modalIsOpen, setModalIsOpen] = useState(false)
+    const [updateDB, setSpdateDB] = useState(false)
+    const [isLoaded, setIsLoaded] = useState(false)
+    const [isError, setIsError] = useState(false)
+
+    function ErrorAlert() {
+        Swal.fire({
+            icon: 'error',
+            title: 'Oops...',
+            text: 'Something went wrong!',
+            footer: '<a href="/">try Again ?</a>'
+        })
+    }
+    useEffect(() => {
+        const getData = async () => {
+            setIsLoaded(true);
+            await axios.get(`http://localhost:3090/accounts`).then((res) => {
+                setAccountDB(res.data)
+                setIsLoaded(false);
+            }).catch((err) => {
+                console.log(err);
+                setIsError(true)
+            });
+        }
+        getData();
+    }, [updateDB])
```

Comment:
this is wrong. You may get infinite call because of this

Reply:
i can't get infinite get request because this state changes when i click "Create new wallet" button so i update the current state , instead i was advised to use (react-query) to make this point with help it ?